src/pages/JoinStep2/JoinStep2.js

```diff
@@ -0,0 +1,365 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./JoinStep2.scss";
+
+class JoinStep2 extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: "",
+      account: "",
+      password: "",
+      email: "",
+      phoneNumber: "",
+      dateOfBirth: 19951116,
+      emailAgreement: false,
+      smsAgreement: false,
+      //여기서부터 백에 전달 안함
+      confirmPw: "",
+      isNameNull: true,
+      isAccountNull: true,
+      isPasswordNull: true,
+      isEmailNull: true,
+      isPhoneNumberNull: true
+    };
+  }
+
+  handleInput = e => {
+    e.preventDefault();
+    const { value, name } = e.target;
+    this.setState({
+      [name]: String(value)
+    });
+    console.log(this.state);
+  };
+
+  handleSMSCheckbox = e => {
+    const { name } = e.target;
+    this.setState({
+      [name]: !this.state.smsAgreement
+    });
+    console.log(this.state);
+  };
+
+  handleEmailCheckbox = e => {
+    const { name } = e.target;
+    this.setState({
+      [name]: !this.state.emailAgreement
+    });
+    console.log(this.state);
+  };
+
+  checkId = () => {
+    const APIforIdCheck = "http://localhost:3000/data/account.json";
+    const { account } = this.state;
+    fetch(APIforIdCheck, {
+      method: "POST",
+      body: JSON.stringify({
+        account: account
```

Comment:
```suggestion
        account
```
키, 벨류 값이 같을 경우 객체 단축 속성명으로 줄여써줄 수 있습니다